src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -169,6 +190,25 @@ const CreateImageWizard = () => {
               >
                 <Gcp />
               </WizardStep>,
+              <WizardStep
+                name="Azure"
+                id="wizard-target-azure"
+                key="wizard-target-azure"
+                footer={
+                  <CustomWizardFooter
+                    disableNext={
+                      azureShareMethod === 'manual'
+                        ? !isAzureTenantGUIDValid(azureTenantId) ||
+                          !isAzureSubscriptionIdValid(azureSubscriptionId) ||
+                          !isAzureResourceGroupValid(azureResourceGroup)
+                        : azureSource === undefined
```

Comment:
When you get to the Azure step for the first time the Next button is enabled even without filling anything in. This should fix it :)

```suggestion
                        : azureSource === ''
```

Reply:
https://github.com/osbuild/image-builder-frontend/assets/73419853/e96c2e0c-37be-49e5-b83e-f5727dfe45ff

I see that its disble

Reply:
It is now! My bad :sweat_smile:

Reply:
no worry :)